ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -64,6 +64,13 @@
       + "will not start serving requests until all preconditions are satisfied. Until that time, "
       + "requests will return a 503 error";
 
+  static final String KSQL_SERVER_ENABLE_UNCAUGHT_EXCEPTION_HANDLER =
+      KSQL_CONFIG_PREFIX + "server.uncaught.exception.handler.enable";
```

Comment:
we should try to name these hierarchically, so the prefix is as general as possible, e.g.:
`server.exception.uncaught.handler.enable`